editor/js/Loader.js

```diff
@@ -828,15 +827,7 @@ function Loader( editor ) {
 
 				loader.parse( data, function ( result ) {
 
-					if ( result.isScene ) {
-
-						editor.execute( new SetSceneCommand( editor, result ) );
```

Comment:
The change makes sense in context of `VRMLLoader`. 

But I'm unsure about `ObjectLoader`. You can export the entire scene to JSON. But when importing it back to the editor, you end up with a different setup (the root scene + the imported scene). I'm not sure this is wanted.

Reply:
TBH, I'm not sure it's a problem when scene user data are overwritten. I would rather accept that than ending up with a different scene hierarchy after the import.

Right now, it feels like the PR solves one issue by introducing another one.

Reply:
The purpose of File>Import is for importing assets to the default scene, not replacing the default scene of any kind.

The more proper action for replacing the default scene,... is File>Open Project (not implemented)

Reply:
I edited the original post to clarify the situation :D

Reply:
#28351 looks good! If @mrdoob likes the save/open pattern, I'm okay with approving this PR as well.